README.md

```diff
@@ -33,8 +33,7 @@ SECRET_KEY=YOURSECRETKEYGOESHERE
 As early as possible in your application bootstrap process, load .env:
 
 ```python
-import os
-from dotenv_vault import load_dotenv
+from dotenv_vault import *
```

Comment:
We shouldn't do this though because it's an API change.

Everyone is already familiar with this api:

```
from dotenv import load_dotenv
load_dotenv()
```

Now we'd be teaching them to do this:

```
from dotenv_vault import *
```

We want them to instead only change a single word in their already written code:

```
from dotenv_vault import load_dotenv
load_dotenv()
```

Reply:
so if there is not a way to import os without changing that api then let's not do it.

Reply:
The ability for a dev to run a string replace on all their projects this way is important for the dev experience. Requiring them to instead poke into code and change the api they are familiar with feels jarring, uncomfortable. I wouldn't personally like it. I'd be annoyed.